src/core_plugins/kibana/ui_setting_defaults.js

```diff
@@ -226,6 +226,12 @@ export function getUiSettingDefaults() {
       value: '($0,0.[00])',
       description: 'Default <a href="http://numeraljs.com/" target="_blank">numeral format</a> for the "currency" format'
     },
+    'format:language': {
+      value: 'en',
+      type: 'select',
+      options: ['be-nl', 'chs', 'cs', 'da-dk', 'de-ch', 'de', 'en', 'en-gb', 'es-ES', 'es', 'et', 'fi', 'fr-CA', 'fr-ch', 'fr', 'hu', 'it', 'ja', 'nl-nl', 'pl', 'pt-br', 'pt-pt', 'ru-UA', 'ru', 'sk', 'th', 'tr', 'uk-UA'],
```

Comment:
It's a shame that we're hard-coding these, but I'm not seeing a way to access all possible languages...

Reply:
@kobelb 
I think exactly like you :)
I'm sad to hard-code these values, but I didn't another way to do that.
That said, this array is likely to change very rarely (maybe never)...